frontend/ui/data/keyboardlayouts/keypopup/el_popup.lua

```diff
@@ -246,15 +325,25 @@ return {
     _Y_el = {
         "Υ",
         northeast = "Ύ",
-        northwest = "Y",
-        north = "Ϋ",
+        north = "Y",
+        west = "Ὑ",
+        south = "Ϋ",
+        southeast = "Ὕ",
+        southwest = "Ὗ",
```

Comment:
@ichnilatis-gr : you had these both associated to `southwest`. I changed one to `southeast`.
Please check as I don't know which of these feels best on the left or on the right.

Greek Capital Letter Upsilon _with Dasia and Oxia_ or _with Dasia and Perispomeni_ (with basil and pepperoni ? :) a bit hard to stomach though :)

Reply:
Thank you for your notice.
However, I would prefer the reverse, to match the order of lowercase letters (as far as _Upsilon with Dasia and Oxia_ is concerned :pizza:).
```suggestion
        southwest = "Ὕ",
        southeast = "Ὗ",
```

Reply:
ok, done

Reply:
👍